app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Here is a list of users</h1>
\ No newline at end of file
+<% @users.each do |user| %>
+  <div>
+    <%= link_to user_path(user) do %>
+      <%= render partial: 'users/user_info', locals: { user: user } %>
```

Comment:
@mershark Great job using partial to render your users, it's a perfect practice that helps the better organization of your code, for easier code reading and scaling in the future.